docs/ERROR_HANDLING.md

```diff
@@ -0,0 +1,70 @@
+# Stark Error Handling
+
+Stark provides its own error handler, StarkErrorHandler, that overrides Angular's ErrorHandler. 
+This handler will dispatch an error which you can treat as you want. 
+
+## Module importation
+
+If you want to use this handler, you simply have to import StarkErrorHandlerModule from stark-core in your `app.module.ts` file.
+
+
+```
+import {StarkErrorHandlerModule} from "@nationalbankbelgium/stark-core";
+
+@NgModule({
+	bootstrap: ...
+	declarations: ...
+	imports: [
+		StarkErrorHandlerModule.forRoot(),
+		...
+		]
+	})
+```
+
+## Effects definition
+
+To define what to do when an error happens, you can simply create a new effects file, like in the following example:
+
+````
```

Comment:
Here you can add the `typescript` keyword so the code is correctly formatted as Typescript code ;)